devicetypes/smartthings/zigbee-switch.src/zigbee-switch.groovy

```diff
@@ -108,6 +108,7 @@ metadata {
 		fingerprint manufacturer: "Jasco Products", model: "43094", deviceJoinName: "Enbrighten Switch" //Enbrighten, Plug-in Smart Switch 43094, Raw Description: 01 0104 0100 00 06 0000 0003 0004 0005 0006 0B05 02 000A 0019
 		fingerprint manufacturer: "Jasco Products", model: "43102", deviceJoinName: "Enbrighten Outlet", ocfDeviceType: "oic.d.smartplug" //Enbrighten, In-Wall Smart Outlet 43102, Raw Description: 01 0104 0100 00 06 0000 0003 0004 0005 0006 0B05 02 000A 0019
 		fingerprint manufacturer: "Jasco Products", model: "43076", deviceJoinName: "Enbrighten Switch" //Enbrighten, In-Wall Smart Switch 43076, Raw Description: 01 0104 0100 00 06 0000 0003 0004 0005 0006 0B05 02 000A 0019
+        fingerprint profileId: "0104", deviceId: "0001", manufacturer: "LELLKI", model: "Switch", deviceJoinName: "LELLKI Switch"
```

Comment:
Please, could You fix indentation here (use 2 tabs) and add a raw description in a comment after the fingerprint (as it's done in above fingerprints) ? 
(Raw Description can be found in device details in graphapi portal).

Reply:
Dear Partner, could you please @KKlimczukS comment and reply? 
Please note that if we do not receive any reply from you in this PR within 1 week, we will need to change status of your request to “failed”.